crates/bin/src/args.rs

```diff
@@ -129,6 +129,19 @@ pub struct Args {
     #[clap(help_heading = "Overrides", long, default_value_t = RateLimit::default())]
     pub rate_limit: RateLimit,
 
+    /// Specify the strategies to be used,
+    /// binstall would run the strategies specified in order.
+    ///
+    /// Default value is "release,quick-install,compile".
+    #[clap(help_heading = "Overrides", long)]
+    pub strategies: Option<Vec<Strategy>>,
```

Comment:
I think we can just do `Vec<Strategy>` and fill in the default if that's empty?

Reply:
Yeah that makes sense, even if user can do `cargo binstall --strategy "" ...` it usually doesn't happen.

Reply:
Done, I've also applied the same to `disable_strategies`.